info.yml

```diff
@@ -24,6 +24,11 @@ configuration:
         default_value: "Loader"
         description: Name to appear on the title of the UI Dialog.
 
+    disable_deprecated_files_actions:
+        type: bool
+        default_value: false
+        description: If loading actions should be disabled on deprecated files.
```

Comment:
I would reword this slightly: "If true, loading actions will be disabled for deprecated files." Something along those lines.